### ISSUE TITLE
horseman.close() not closing

### ISSUE THREAD

**Issue by dzoba** [johntitus/node-horseman#39]
After executing a file that uses horseman to load a page, the file never completes executing.  This is the code I'm using:

```
var Horseman = require("node-horseman");
var horseman = new Horseman();

horseman
  .userAgent("Mozilla/5.0 (Windows NT 6.1; WOW64; rv:27.0) Gecko/20100101 Firefox/27.0")
  .open('http://www.google.com')
  .close();
```

I then run ```node horsemantest.js``` and it just hangs.

I am using node 0.12.5 and phantomjs 2.0.0, and the latest node-horseman.

**Comment by johntitus** [comment]
@dzoba - Thank's for finding this. It appears to be a polling issue in `node-phantom-simple`. There is a pull request against that project with a fix. If you need a fix now, I suggest grabbing it from the linked issue (#69).  I'll update `node-horseman` as soon as `node-phantom-simple` implements the fix.

**Comment by johntitus** [comment]
Seems resolved. Published version 1.5.6.
